src/registrar/views/domain_request.py

```diff
@@ -842,18 +825,16 @@ def _get_orphaned_contacts(self, domain_request: DomainRequest, check_db=False):
 
         # Get each contact object on the DomainRequest object
         so = domain_request.senior_official
-        submitter = domain_request.submitter
         other_contacts = list(domain_request.other_contacts.all())
         other_contact_ids = domain_request.other_contacts.all().values_list("id", flat=True)
 
         # Check if the desired item still exists in the DB
         if check_db:
             so = self._get_contacts_by_id([so.id]).first() if so is not None else None
-            submitter = self._get_contacts_by_id([submitter.id]).first() if submitter is not None else None
             other_contacts = self._get_contacts_by_id(other_contact_ids)
 
         # Pair each contact with its db related name for use in checking if it has joins
-        checked_contacts = [(so, "senior_official"), (submitter, "submitted_domain_requests")]
```

Comment:
We do not replace submitter with creator here since creator is a contact that exists isolated from the domain request